test/unit/sort.js

```diff
@@ -0,0 +1,55 @@
+import chai from 'chai';
+import { StreamChat } from '../../src';
+
+const expect = chai.expect;
+
+describe('test if sort is deterministic', () => {
```

Comment:
can we move this test to the client test file?

Reply:
You mean, rename it to `client.js`? I don't see any client-related js files right now. There's `channel_state.js` and a bunch of integration tests

Reply:
you're right, `client.js` test file is not yet merged to master. let's leave it as it is and I will move the tests there later.